pkg/scheduler/cache/cache.go

```diff
@@ -300,7 +301,22 @@ func (su *defaultStatusUpdater) UpdatePodGroup(pg *schedulingapi.PodGroup) (*sch
 		return nil, err
 	}
 
-	updated, err := su.vcclient.SchedulingV1beta1().PodGroups(podgroup.Namespace).Update(context.TODO(), podgroup, metav1.UpdateOptions{})
```

Comment:
I'm not sure if this modification is correct. I remember that this function will also update the annotation and label, so it is wrong to only update the status.

Reply:
> we may rewrite the logic , not to persist these nums fields in podgroups, like this pr: https://github.com/volcano-sh/volcano/pull/3751
I think the proposal is want to not update the status fields in podgroups, do you mean the annotation and labels still need to be updated?